### PR TITLE
Revert gCNV WDLs to tar calls from all samples.

### LINKED ISSUE #5217: gCNV WGS cohort germline WDL run outputs vcf files with names that do not correspond to the actual sample inside the file.
I used the very last version of the WDL using the last master commit for the docker and the name of the output files seem to be a random permutation of the actual sample names. The name inside the file (i.e. the one listed in the #CHROM line) seems to be correct.

A previous run using a earlier version of the WDL (probably the one just before the last update) and the latest official gatk docker didn't have this issue.
.

### PR DESCRIPTION
**Pull request by samuelklee**
Closes #5217.

@vruano, I've verified that cohort and scattered-case mode WDLs run correctly on FC (to be precise, I tested the WDLs from my sl_filter branch rebased on this branch, but this branch alone should be fine as well).

However, #4397 should still be addressed at some point in the future.  This can wait until we have v1 of the FC evaluation in place.

### CODE REVIEW

File: scripts/cnv_wdl/cnv_common_tasks.wdl [hunk 1]
```diff
@@ -287,8 +287,8 @@ task PostprocessGermlineCNVCalls {
         calls_args=""
         for index in ${dollar}{!gcnv_calls_tar_array[@]}; do
             gcnv_calls_tar=${dollar}{gcnv_calls_tar_array[$index]}
-            mkdir -p CALLS_$index/SAMPLE_${sample_index}
-            tar xzf $gcnv_calls_tar -C CALLS_$index/SAMPLE_${sample_index}
+            mkdir CALLS_$index
+            tar xzf $gcnv_calls_tar -C CALLS_$index
             cp ${dollar}{calling_configs_array[$index]} CALLS_$index/
```
Comment (asmirnov239) [inline scripts/cnv_wdl/cnv_common_tasks.wdl]: You don't need to pass as input or untar any of these anymore:
 ```
    Array[File] calling_configs
    Array[File] denoising_configs
    Array[File] gcnvkernel_version
    Array[File] sharded_interval_lists
```
Look at the diff for `cnv_common_tasks.wdl` https://github.com/broadinstitute/gatk/commit/f0dddbc686a87cf969f4be5b0594edda081690c1#diff-f3a83ecdd107a7c42aee06dc41842fd2

Comment (samuelklee) [reply scripts/cnv_wdl/cnv_common_tasks.wdl]: Thanks for catching this.  I removed them from the PostprocessGermlineCNVCalls task but left them as outputs of the workflows.  Might be useful for debugging, but we can always change this later if you like.

Comment (samuelklee) [reply scripts/cnv_wdl/cnv_common_tasks.wdl]: I also added the counts as outputs of the scattered case workflow.  Perhaps at some point we can flatten the outputs there.  This is trivial for all outputs except the ploidy calls, which need to be uncompressed, concatenated, and compressed again.

**Comment by samuelklee** [comment]
Thanks, back to you @asmirnov239.

**Pull request merged**